### PR TITLE
ws_eink: Add bitrev8 to ws_eink_display for issue mirror with littlevgl

### PR DESCRIPTION
**Pull request by nxthongbk**
Add bitrev8 to ws_eink_display for issue mirror with littlevgl graphic lib.

### CODE REVIEW

**Comment by dpfrey** [comment]
Hi Thong,

This pull request still hasn't been submitted properly.  Notice that there are 4 commits listed in the pull request.  There should be only one.  The commit branch submitted as a pull request should not have any unrelated ancestor commits that aren't part of the upstream repository.  Please rebase your commit onto mangOH's master branch and resubmit.

File: experimental/waveshare_eink/linux_kernel_modules/fb_waveshare_eink.c [hunk 1]
```diff
@@ -359,6 +361,12 @@ static int ws_eink_update_display(struct ws_eink_fb_par *par)
 		return ret;
 
 	memcpy(&ssbuf, &vmem, sizeof(vmem));
```
Comment (dpfrey) [inline experimental/waveshare_eink/linux_kernel_modules/fb_waveshare_eink.c]: This is just a weird way of doing:
`ssbuf = vmem;`

File: experimental/waveshare_eink/linux_kernel_modules/fb_waveshare_eink.c [hunk 2]
```diff
@@ -359,6 +361,12 @@ static int ws_eink_update_display(struct ws_eink_fb_par *par)
...
+
```
Comment (dpfrey) [inline experimental/waveshare_eink/linux_kernel_modules/fb_waveshare_eink.c]: This is wrong. `ssbuf`, `vmem` and `par->info->screen_base` are all just pointers to the same chunk of memory.  What we need to do is make a copy of the data in `par->info->screen_base` and then call `bitrev8` on every byte of it.  What this code is doing is calling `bitrev8` on the original data.  I suggest allocating another block of memory at the time that vmem is allocated in probe.  Then you can store a pointer to the memory in `struct ws_eink_fb_par`

**Comment by nxthongbk** [comment]
Hi David.

Okie I rebased onto mangOH's master branch and create new branch to for new pull request #24.

So I closed this pull request.

Thanks
Thong Nguyen

**Pull request closed without merging**